Mathlib/RingTheory/Localization/AtMinimalPrime.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2024 Uni Marx. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Uni Marx
+-/
+import Mathlib.RingTheory.Localization.AtPrime
```

Comment:
I think this whole file should go inside `Localization.AtPrime` which is still quite small.

Reply:
Not possible due to import cycles, sadly. I could put it in Ideal.MinimalPrime but I don't think that fits at all

Reply:
I have looked at `ideal.MinimalPrime` and I think it would indeed fit there: there are results concerning localization (which are precisely the ones causing this import clash) and yours would be part of those.